### DISCUSSION TITLE
What happened to `to_index()` in PythonObject?

### DISCUSSION THREAD

**Discussion by gautam-e** [modular/modular#405] [Q&A]
I could've sworn I saw `to_index` in the PythonObject module. But it isn't there anymore. 
I can't seem to find it via the search function either (apart from in the Notebooks). Was it removed? I didn't see it in the Changelog too.

**Comment by mojodojodev** [comment]
It got moved to the dunder method `__index__()`:

```rust
let x = Python.evaluate("1 + 2")

print(x.__index__())
```
```
3
```

**Reply by gautam-e** [reply]
Thanks @mojodojodev I guess I'll make an issue for it since the notebooks need to be updated then.
